gl2/khrplatform.h

```diff
@@ -0,0 +1,282 @@
+#ifndef __khrplatform_h_
```

Comment:
What is this file for?

Reply:
"The Khronos group maintains several industry standard APIs: https://www.khronos.org/registry/

To keep things simpler Khronos maintains the khrplatform header to be used by other headers maintained by Khronos, defining common data types, tokens and calling convention macros." -- Some random guy on SO.

Basically, it's just a header for OpenGL datatypes or macros that we may or may not need. I'll remove it since it doesn't seem necessary right now.